src/redux/reducers/profileReducer.js

```diff
@@ -1,4 +1,5 @@
-import { ADD_POST, UPDATE_NEW_POST_TEXT } from './actions/actionTypes';
+import { ADD_POST, UPDATE_NEW_POST_TEXT } from '../actions/actionTypes';
+import { DEFAULT_LIKES_VALUE, NEXT_ID } from '../../utils/constants';
```

Comment:
Consider moving this import.
I already asked your to reorganize the order of imports several times.

Reply:
/fxd